packages/alea-frontend/pages/api/get-students-enrolled-in-course.ts

```diff
@@ -0,0 +1,24 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { executeAndEndSet500OnError } from './comment-utils';
+
+const ENROLLMENTS = 'enrollments';
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  try {
+    const { courseId, instanceId } = req.body;
+    if (!courseId || !instanceId) {
+      return res.status(400).json({ error: 'Missing courseId or instanceId in request body.' });
+    }
+    const aclId = `${courseId}-${instanceId}-${ENROLLMENTS}`;
```

Comment:
Create utils function that takes in courseId and instanceId gets the course enrollments acl name. 
Reuse the same function in `packages/alea-frontend/pages/course-home/[courseId].tsx`